template-processors/base/bin/processTemplates.sh

```diff
@@ -18,4 +18,5 @@ set -euxo pipefail
 
 echo Processing Templates
 
+# shellcheck disable=SC2086
 cp -R $CLONED_TEMPLATE_GIT_DIR/* $MANIFEST_DIR/
```

Comment:
Please change this line to:

    cp -R "$CLONED_TEMPLATE_GIT_DIR"/* "$MANIFEST_DIR/"

Reply:
This is breaking the changes of e2e tests. I can add this in comment